src/GrpcProtos/capture.proto

```diff
@@ -599,6 +608,8 @@ message ThreadStateSlice {
   // Otherwise, they should be 0 to minimize bandwidth usage.
   uint32 wakeup_tid = 7;
   uint32 wakeup_pid = 8;
+
+  uint64 callstack_id = 10;
```

Comment:
I think the name or a comment should clarify that this is the callstack "at the end" of the slice (right?).